packages/nimble-components/src/table/index.ts

```diff
@@ -135,6 +148,7 @@ export class Table<
         super.connectedCallback();
         this.virtualizer.connectedCallback();
         this.validateAndObserveColumns();
+        this.viewport.addEventListener('scroll', this.onViewPortScroll);
```

Comment:
connectedCallback can run multiple times if an element is temporarily removed and readded to the dom, so this would need clean-up in disconnected.

Instead we should register this event in the FAST template as a [passive scroll listener](https://developer.chrome.com/en/docs/lighthouse/best-practices/uses-passive-event-listeners/#how-to-make-event-listeners-passive-to-improve-scrolling-performance) since we don't require preventDefault.

Edit: okay actually FAST vNext has the abilitiy to [register passive event listeners from a template binding](https://github.com/microsoft/fast/issues/4531#issuecomment-963323586).

So instead we should make this event listener passive and make sure we clean-up in the disconnectedCallback